Content.Shared/Trigger/Components/Triggers/TriggerOnBuckledComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Trigger.Components.Triggers;
+
+/// <summary>
+/// Triggers when the component parent is buckled.
+/// This is intended to be used on buckle-able entities like mobs.
+/// The parent object should be the entity that is being buckled to something else.
```

Comment:
```suggestion
/// Triggers when the owning entity is buckled.
/// This is intended to be used on buckle-able entities like mobs.
/// The user is the strap entity (a chair or similar).
```
The name "parent" is a little unclear, components usually don't have parents. This component's class does however have the `BaseTriggerOnXComponent` parent, which makes this a little confusing.
And always mention what the user for the trigger is.